<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetClass</class>
 <widget class="QWidget" name="WidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="getExistingDirectory">
     <property name="text">
      <string>getExistingDirectory</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="getOpenFileName">
     <property name="text">
      <string>getOpenFileName</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="getOpenFileNames">
     <property name="text">
      <string>getOpenFileNames</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="getSaveFileName">
     <property name="text">
      <string>getSaveFileName</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Widget.qrc"/>
 </resources>
 <connections/>
</ui>
